run-network/creator.py

```diff
@@ -37,13 +37,17 @@ def create_env_cli(config_path: str):
 
 @cli.command()
 @click.argument("config_path")
-def start(config_path):
+@click.argument("profile", default="minimum", type=click.Choice(["minimum", "medium", "maximum"]))
+def start(config_path, profile):
     """Start docker-compose in base_dir"""
     config: Config = Config.from_file(config_path)
     dest = Path(config.dest)
     chdir(dest.parent)
     environ["PWD"] = str(dest.parent)
-    command = ["docker-compose", "-f", f"{dest.name}", "up", "-d"]
+    command = ["docker-compose", "-f", f"{dest.name}"]
+    if profile:
+        command += ["--profile", profile]
+    command += ["up", "-d"]
     subprocess.run(command)
```

Comment:
IINM, here it would simply be:
```suggestion
    command += ["up", "-d"]
    subprocess.run(command)
```

So we don't even need a `env={"COMPOSE_PROFILES": os.getenv("COMPOSE_PROFILES")}` for run, because the shell (and Python) will simply inherit this. Python's `subprocess.Popen` docs say:
>  If `env` is not None, it must be a mapping that defines the environment variables for the new process; these are used instead of inheriting the current process’ environment, which is the default behavior.

Have not tested, though.